ptp/c4u/clock/clock.go

```diff
@@ -27,20 +32,25 @@ const (
 	ClockClassUncalibrated ptp.ClockClass = ptp.ClockClass52
 )
 
+type DataPoint struct {
+	PHCOffset         time.Duration
+	OscillatordStatus *osc.Status
```

Comment:
Can we please define a custom type (offset + clockClass) and populate it in oscillatord() instead of throwing around unnecessary data which will change a format very soon.